elliott/elliottlib/cli/snapshot_cli.py

```diff
@@ -46,25 +59,27 @@ async def _get(nvrs) -> list[KonfluxRecord]:
             raise e
         return records
 
-    nvr_record_map = {}
-    records = []
-    bundle_nvrs = [n for n in nvrs if "-bundle-container" in n]
-    non_bundle_nvrs = [n for n in nvrs if n not in bundle_nvrs]
-
-    if bundle_nvrs:
-        runtime.konflux_db.bind(KonfluxBundleBuildRecord)
-        bundle_records = await _get(bundle_nvrs)
-        records.extend(bundle_records)
-    if non_bundle_nvrs:
-        if for_fbc:
-            runtime.konflux_db.bind(KonfluxFbcBuildRecord)
+    # Group NVRs by their type to fetch them in parallel
+    type_nvrs = defaultdict(list)
+    for nvr in nvrs:
+        nvr_dict = parse_nvr(nvr)
+        if nvr_dict['name'].endswith('-bundle-container'):
```

Comment:
The bundle name could also use `-metadata-container`

Reply:
Good point @Ximinhan ! I will check and add it in the next push.